acceptance-tests/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>acceptance-tests</artifactId>
+    <name>Bitbucket Server Integration Plugin Acceptance Tests</name>
+    <description>Includes the integration/acceptance tests for the Bitbucket Server Jenkins integration plugin</description>
+    <packaging>jar</packaging>
+    <version>1.1.1-SNAPSHOT</version>
+
+    <properties>
+        <jenkins.version>2.176.1</jenkins.version>
+        <scribejava.version>6.8.1</scribejava.version>
+        <jackson.version>2.10.3</jackson.version>
+        <groovy.version>2.4.12</groovy.version>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Java version should be driven by parent pom which should apply globally to the plugin as well as the tests. We have tests on Java 11 as well.

Reply:
Replaced with `<java.level>8</java.level>` for now to match the plugin pom (until we have a proper parent pom, if/when we can figure out how to do that).

Reply:
Nope, doesn't work:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
```
It doesn't seem to be inheriting the Java source/target version from anywhere, so unfortunately looks like I need to add it back here.

Reply:
> We have tests on Java 11 as well.

AFAIK, that's unrelated to `maven.compiler.source` and `maven.compiler.target`; these two variables mean that we're using Java language constructs from Java 8, and not higher (e.g. `var`, etc.). Tests should still be able to run in Java 11 or any other version regardless.